com.hybris.hyeclipse.utils/src/com/hybris/hyeclipse/testers/AbstractFilePropertyTester.java

```diff
@@ -48,13 +47,20 @@ protected boolean testSelectedFileByExtensions(final Object receiver, final Set<
 				return fileExtensions.contains(file.getFileExtension());
 			}
 		} else if (receiver instanceof List) {
+			boolean doesExtensionMatch = false;
```

Comment:
Do you expect more than one file? If yes please update javadoc + please use single return, also consider using lambda